scripts/build/termux_step_install_license.sh

```diff
@@ -53,36 +53,41 @@ termux_step_install_license() {
 				|'curl'|'HPND'|'ISC'|'Libpng'|'MIT'|'Openfont-1.1'\
 				|'PythonPL'|'X11'|'ZLIB')
 				# Find the license file in the source files
+				local FOUND=0
```

Comment:
And... I just thought of another thing.
We only want the "specific" license case to run once.
Since it loops through the common license names it'll add all the license files it picks up in a single run.
Running it again, if we had say `ISC, MIT, BSD` as a license combination would duplicate all the license files.

Reply:
Yeah, that too. There are two issues here.

1. Multiple common license files will get added as we don't break out of loop when a common license is found like I said above. Like if they were both `copyright` and `license` files in source repo, they would both get added.
2. If `TERMUX_PKG_LICENSE` has multiple licenses that will use common licenses, duplicates would get added.

I am not sure if `1` is a real issue or not and if we should be doing that. But for `2`, a bash associative array could be used to add basenames for both generic and non-generic licenses and skipped if already added.

Reply:
<details><summary>I built a test case for this.</summary>
<p>

![image](https://github.com/user-attachments/assets/56bb22f5-f939-470d-bb8e-5c6289f67d4d)



</p>
</details> 

I modified `gtkwave`'s build script to have `GPL-2.0, MIT, BSD, ISC` as the licenses.
It should be installing:
- A symlink for GPL-2.0
- `COPYING` from the source files
- `LICENSE.TXT` from the source files

3 licenses total

<details><summary>It actually includes 7.</summary>
<p>

![image](https://github.com/user-attachments/assets/68360e73-b6aa-4688-abc3-28884c99916e)

</p>
</details>

Incidentally, I also updated our `gtkwave` package to use the github repo as a source instead of the outdated sourceforge repo and build against GTK3 instead of GTK2.

Reply:
Since multiple common licenses can exist, just use a bash associative array for basenames like done for `TERMUX_PKG_LICENSE_FILE` and skip adding. Generic licenses should not conflict with common license basenames, but would prevent duplicates to be added if same generic license is specified multiple times in `TERMUX_PKG_LICENSE` and build can be aborted with error.

Reply:
I managed to co-opt the `FOUND` variable for this instead.
I've renamed it `FROM_SOURCES` now.
The `for` loop over the common license files only needs to run if our license files included `FROM_SOURCES` is still 0.

Reply:
That should work too.

Reply:
Success.
![image](https://github.com/user-attachments/assets/6aa8c5f5-aaec-4bf7-9d05-4af7334aa1f5)

Reply:
Congratz! 🎉

Reply:
I'll be sending a separate PR for the `gtkwave` update as well.
Though it will be depending on the changes in this PR.

Also we'll need to add an Epoch to the package, the sourceforge and github release numbers diverge.
And all other packages of it ship the GitHub release.